docs/src/ScrollableContainer.doc.js

```diff
@@ -0,0 +1,593 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PageHeader from './components/PageHeader.js';
+import PropTable from './components/PropTable.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="ScrollableContainer"
+    description="ScrollableContainer is used with anchored, hovering components such Flyout, Tooltip, Dropdown, or Typeahead. This is needed for proper positioning when the anchor component could scroll within the viewport."
+  />,
+);
+
+card(
+  <MainSection name="Examples">
```

Comment:
I would move props above this, and rename this to Variants. I think for some of these more primitive components, we can skip the top-most example

Reply:
Agreed.

Reply:
DONE